packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
i'm asking why, at this point, is it both in `this.state` and directly on `this`? it seems to me that `this.locator` should be sufficient (and preferable, as it never changes) _everywhere_.